src/DDL/0_geom.sql

```diff
@@ -1,9 +1,9 @@
 CREATE TABLE geom
 (
-	    geom_record_id integer primary key,
```

Comment:
Used serial as the datatype here, which will be inherited by child tables so that they all share the same sequence.